server/src/models/job_posts.model.js

```diff
@@ -44,15 +44,31 @@ const JobPosting = sequelize.define(
         "Weekly",
         "Annually",
         "Commission",
-        "Base & Commission",
+        "Base Commission", // Changed for MySQL compatibility
```

Comment:
Can you remove this comment?

Reply:
Just removed the comment in commit c31430e! Hope that helps.